.github/workflows/macos.yml

```diff
@@ -18,4 +18,4 @@ jobs:
       - name: Install Dependencies
         run: gem install "rake:>=10.4.2" "minitest:>=5.4.3" --conservative --no-document
       - name: Run Test
-        run: rake test
+        run: rake TEST=test/rubygems/test_build_commands.rb
```

Comment:
Why we not running all the tests in Mac OS?

Reply:
Just to save some CI time for debugging. I'll restore all tests later.